chapter_5/exercise_5_06/getline.c

```diff
@@ -1,23 +1,19 @@
 #include <stdio.h>
 
-void get_line(char *s);
 
-int main(void)
-{
-  char string[150] = "";
+char *getln(char *line);
```

Comment:
@kyopa First I want to thank you for your interest in improving this solution. I will leave some comments here and there so that you know what I also think about your proposal.